src/compiler/program.ts

```diff
@@ -559,6 +559,12 @@ namespace ts {
         sourceMap: false,
     };
 
+    interface OutputFingerprint {

```

Comment:
@DanielRosenwasser I am confused, it's not even exported, why `/** @internal */` is needed?